src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -141,6 +144,28 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         return tagSet;
     }
 
+    /**
+     * Parses {@code String date} into an {@code LocalDate} and returns it. Leading and trailing whitespaces will be
+     * trimmed.
+     */
+    public static LocalDate parseDate(String date) throws ParseException {
+        requireNonNull(date);
+        String trimmedDate = date.trim();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        return LocalDate.parse(trimmedDate, formatter);
+    }
+
+    /**
+     * Parses {@code String time} into an {@code LocalTime} and returns it. Leading and trailing whitespaces will be
+     * trimmed.
+     */
+    public static LocalTime parseTime(String time) throws ParseException {
```

Comment:
here also I think should check if valid localtime string before trying to parse it as a LocalTime